windows/src/buildtools/sentrytool/Keyman.System.SentryTool.SentryToolMain.pas

```diff
@@ -93,7 +93,7 @@ function FindFilePath(const UnitName: string): string;
     else Result := FDelphiSearchFile.FindFile(UnitName);
 
   if Result <> '' then
-    Result := ExtractRelativePath(FRootPath, Result);
+    Result := ExpandFileName(Result); // ExtractRelativePath(FRootPath, Result);
```

Comment:
Full path allows the debug symbol parsing to associate with source files. However, I am waiting to see what happens with GitHub linking shortly; it may need adaptation yet.